### PR TITLE
Wamartin/add py311 support

### PR DESCRIPTION
**Pull request by wamartin-aml**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
     'Programming Language :: Python :: 3.8',
     'Programming Language :: Python :: 3.9',
     'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
```
Comment (trangevi) [inline setup.py:36]: I think we need to update line 53 as well

File: setup.py [hunk 2]
```diff
@@ -33,6 +33,7 @@
     'Programming Language :: Python :: 3.8',
```
Comment (trangevi) [inline setup.py:33]: Should we remove 3.8 you think?

**Review by trangevi — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (wamartin-aml) [reply setup.py:33]: I think we can wait until the inference server drops support maybe?

File: setup.py (hunk 1 above)
Comment (wamartin-aml) [reply setup.py:36]: updated

**Review by trangevi — DISMISSED** [review]

**Review by trangevi — APPROVED** [review]

**Pull request merged**